apps/engineeringblog/components/NavBarNavList/NavBarNavList.tsx

```diff
@@ -24,23 +24,33 @@ interface SocialMediaLink {
   url: string;
 }
 
-interface Props {
+interface NavBarNavListProps {
+  NavListItemLinkProps?: LinkProps;
   NavListItemProps?: NavListItemProps;
-  direction?: string;
+  direction?: string | object;
```

Comment:
I think it would be much better if we add context about `object` type

Reply:
_give someone an inch and they'll take a mile_ ... 👍🏽